src/commands/install-geckodriver.yml

```diff
@@ -98,7 +88,7 @@ steps:
 
         GECKODRIVER_VERSION_NUM="$(echo $GECKODRIVER_VERSION | sed -E 's/v//')"
 
-        if geckodriver --version | grep "$GECKODRIVER_VERSION_NUM"<<^parameters.debug>> > /dev/null 2>&1<</parameters.debug>>; then
+        if geckodriver --version | grep "$GECKODRIVER_VERSION_NU > /dev/null 2>&1; then
```

Comment:
> "$GECKODRIVER_VERSION_NU

Looks like this was cut too short. I think it's suppose to be:

`"$GECKODRIVER_VERSION_NUM"`

Reply:
Thank you, I'll have to dig in deeper as well to find out why that didn't seemingly break anything.

Reply:
Geckdriver testing was disabled previously because it was broken on alpine.